koku/subs/trino_sql/azure/subs_row_count.sql

```diff
@@ -10,13 +10,13 @@ WHERE
     AND json_extract_scalar(lower(additionalinfo), '$.vcpus') IS NOT NULL
     AND json_extract_scalar(lower(lower(tags)), '$.com_redhat_rhel') IS NOT NULL
     -- ensure there is usage
-    AND ceil(coalesce(nullif(quantity, 0), usagequantity)) > 0
+    AND ceil(nullif(quantity, 0), 0) > 0
```

Comment:
```suggestion
    AND ceil(quantity, 0) > 0
```
not sure we need the nullif here.